app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,17 @@
+class TasksController < ApplicationController
+  def create
+    @task = current_user.tasks.create(task_params)
```

Comment:
what if there is an error while creating task. Please add success/failure messages.